src/wms/src/test/java/org/geoserver/wms/WMSTest.java

```diff
@@ -75,9 +81,35 @@ protected void setupStartEndTimeDimension(
         getCatalog().save(info);
     }
 
+    /**
+     * set up a feature dimension, but additionally allow to set a default value setting
```

Comment:
Same as in the other PR, start sentences with a capital

Reply:
Done.